shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -133,15 +131,14 @@ class PlatformViewIOS final : public PlatformView {
     std::function<void(bool)> set_semantics_enabled_;
   };
 
-  fml::WeakNSObject<FlutterViewController> owner_controller_;
+  __weak FlutterViewController* owner_controller_;
```

Comment:
Let's rename to `ownerController`

Reply:
Since it's a field on a C++ object, the linter is going to want it to have a C++ field name, even though it looks horrible at the call sites.

Reply:
It's also the style rule: https://google.github.io/styleguide/objcguide.html#style-matches-the-language